### ISSUE TITLE
Record(s) for (ዜና፡) ሕማማቲሁ፡ ለክርስቶስ፡

### ISSUE THREAD

**Issue by smaugustine** [BetaMasaheft/Documentation#1501]
As discussed by Pierluigi Piovanelli in his essay "The _Story of the Passion of Christ_: A New Ethiopic Apocryphon..." (in the memorial volume for Paolo Marrassini), MS BnF Éth 86 preserves a Gǝʿǝz translation of a Latin devotional which begins "ክሡታት፡ ሕማማተ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ... ዘከሠተ፡ ሎን፡ ለሠላስ፡ ቅድሳት፡ ..." and names Brigitte (ብረዜዳ፡), Matilda (ሜቴልዳ፡), and Elizabeth (ኤልሳቤጥ፡). This was later condensed and revised into a short prayer usually appended to MSS of ልፋፈ፡ ጽድቅ፡ which simply begins "ዜና፡ ሕማማቲሁ፡ ለክርስቶስ፡" and names Mary Magdalene, Salome, and Elizabeth.

It seems that there isn't a work record for either of these. There is, notably, a record (LIT5020Hemamatihu [@thea-m]) for what appears to be an intermediate version, unknown to Piovanelli, which is of a similar length to the former but, like the latter, changes the names to Mary, Salome, and Elizabeth.

What would be the best approach for having work records for these two (three?) recensions of this text?

**Comment by thea-m** [comment]
https://github.com/BetaMasaheft/Documentation/issues/1500

**Comment by thea-m** [comment]
Very interesting! My modest proposal after a superficial reading of Piovanelli would be to create 1) an ID for the shorter version as edited by him from several MSS, 2) an ID for the version in BNFet86, and 3) keep the existing LIT5020Hemamatihu as a general ID to be used in cases where the recension is not known (my guess would be that the two BL records that refer it contain the shorter version, but we have no images). What do you think? 
That is without taking into consideration the text posted in https://github.com/BetaMasaheft/Documentation/issues/1500. @DenisNosnitsin1970 , did you have a closer look at it?

Of course, as a student of Marburg University, I am delighted to find St. Elisabeth on BM!

**Comment by smaugustine** [comment]
Just to add, [the images for BL Or 631 are online](http://www.bl.uk/manuscripts/Viewer.aspx?ref=or_631_f002v), but I only skimmed the beginning. It seems to be the long recension but with the changed names.

**Comment by DenisNosnitsin1970** [comment]
Yes, this is one of the Hemamatihu presented by Piovanelli, but EMIP 811 seems to be different. You are now linked and can have a look at it, I am retrieveing the full ms EMIP 811.

**Comment by thea-m** [comment]
@DenisNosnitsin1970 , I assigned you to monitor the closing of issue because you had volunteered to take on the related https://github.com/BetaMasaheft/Documentation/issues/1500

**Comment by DenisNosnitsin1970** [comment]
It is already completed and can be closed.

**Comment by smaugustine** [comment]
Just a follow-up question, did a record for the Piovanelli-type ዜና፡ ሕማማቲሁ፡ ለክርስቶስ፡ ever get created?

**Comment by thea-m** [comment]
Sorry for being so late! I don't think it was. @DenisNosnitsin1970 ?
If Denis has no time for this now, I can do it in the next days (waiting a bit until I get the article delivered, to which I don't have access now...)

**Comment by DenisNosnitsin1970** [comment]
It is now remote in time and i cannot remember if I did this or did not;  I created at least one, LIT6193HomiliaryPassions.

**Comment by thea-m** [comment]
Having reread the material now, as I understand it, LIT6193HomiliaryPassions was created for the homiliary attested in EMIP 812, BNF Ethiopien 116 and Abbadie 29 and is a different text from the shorter Zenā ḥəmāmātihu texts edited by Piovanelli. 
I would actually stick with what I proposed months ago, to create: create 1) an ID for the shorter version as edited by Piovanelli from several MSS, 2) an ID for the version in BNFet86, and 3) keep the existing LIT5020Hemamatihu as a general ID to be used in cases where the recension is not known. BL Or 631 indeed seems to have the text of the longer version attested in BNFet86, but with the names and an ending as in the short versions. It seems reasonable to me to keep using the soon-to-be general ID LIT5020Hemamatihu for this.
If there are no objections over the course of next week, I'll carry out these changes.

**Comment by eu-genia** [comment]
Have the proposed records been created? Just checking

**Comment by eu-genia** [comment]
Just to get some clarity - we now have

LIT5020Hemamatihu - general
LIT6193HomiliaryPassions - BNFet116, EMIP812

and also
LIT6618DersanaMadhane - general
(also LIT6616HomilyPassionIntro; LIT6617PassionsHomilyMonday)
LIT6619HomiliesFeastSaviour

is this OK? which ones are
1) an ID for the shorter version as edited by Piovanelli from several MSS?
2) an ID for the version in BNFet86?

@thea-m @DenisNosnitsin1970

**Comment by eu-genia** [comment]
@thea-m maybe the ID for BNFet86 is still lacking? I do not see anything in the ms...

**Comment by eu-genia** [comment]
can this be closed? @thea-m can you check?

**Comment by thea-m** [comment]
I'm so sorry, I do not currently have the bandwith to get back into this topic right now.
